ceph/basic-ceph-pjdfs.sh

```diff
@@ -0,0 +1,277 @@
+#!/bin/sh
+
+# if any command fails, the script should exit
+set -e
+
+# enable some more output
+set -x
+
+# these variables need to be set
+[ -n "${GERRIT_HOST}" ]
+[ -n "${GERRIT_PROJECT}" ]
+[ -n "${GERRIT_REFSPEC}" ]
```

Comment:
This being a weekly test do we need to checkout `github.com` rather than `gerrithub`.